app/search/fetchSearchResult.server.ts

```diff
@@ -6,6 +6,14 @@ import {
   useSearchWithPaginationQuery,
 } from "@/lib/graphql/generated/fbi/graphql"
 
+const facetTypes = [
+  "materialTypesGeneral",
+  "mainLanguages",
+  "age",
+  "lix",
+  "subjects",
+] as FacetField[]
```

Comment:
Great to have them defined only once. In the future (when the config setup is ready) it should be defined there, I think.

Reply:
agree